src/Sylius/Bundle/ShopBundle/Resources/config/app/config.yml

```
@@ -312,3 +216,900 @@ sylius_ui:
+
+        sylius.shop.layout.footer:
+            blocks:
+                grid:
+                    template: "@SyliusShop/Layout/Footer/_grid.html.twig"
+                    priority: 20
+                content:
+                    template: "@SyliusShop/Layout/Footer/_content.html.twig"
+                    priority: 10
+                after_legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    priority: 5
+                    context:
+                        event: sylius.shop.layout.after_footer
+                        
+        sylius.shop.layout.footer.grid:
+            blocks:
+                before_legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    priority: 45
+                    context:
+                        event: sylius.shop.layout.before_footer
+                plus:
+                    template: "@SyliusShop/Layout/Footer/Grid/_plus.html.twig"
+                    priority: 40
+                your_store:
+                    template: "@SyliusShop/Layout/Footer/Grid/_your_store.html.twig"
+                    priority: 30
+                customer_care:
+                    template: "@SyliusShop/Layout/Footer/Grid/_customer_care.html.twig"
+                    priority: 20
+                payment_methods:
+                    template: "@SyliusShop/Layout/Footer/Grid/_payment_methods.html.twig"
+                    priority: 10
+
+        sylius.shop.layout.header.grid:
+            blocks:
+                logo:
+                    template: "@SyliusShop/Layout/Header/_logo.html.twig"
+                    priority: 30
+                content:
+                    template: "@SyliusShop/Layout/Header/_content.html.twig"
+                    priority: 20
+                cart:
+                    template: "@SyliusShop/Layout/Header/_cart.html.twig"
+                    priority: 10
+
+        sylius.shop.layout.header.content:
+            blocks:
+                legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    context:
+                        event: sylius.shop.layout.header
+                        
+        sylius.shop.login.register_column:
+            blocks:
+                before_legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    priority: 15
+                    context:
+                        event: sylius.shop.login.before_register
+                content:
+                    template: "@SyliusShop/Login/_register.html.twig"
+                    priority: 10
+                after_legacy:
+                    template: "@SyliusUi/Block/_legacySonataEvent.html.twig"
+                    priority: 5
+                    context:
+                        event: sylius.shop.login.after_register
+                        
+        sylius.shop.login.login_column:
```

`login` word is duplicate, maybe is possible to change it
```suggestion
        sylius.shop.login.main_column:
```